.prettierrc

```
@@ -0,0 +1,11 @@
+{
+  "printWidth": 100,
+  "arrowParens": "avoid",
+  "bracketSpacing": true,
+  "singleQuote": true,
+  "semi": false,
+  "trailingComma": "all",
```

Can you remove comma dangle? We don't use it in our repos
```suggestion
  "trailingComma": "none",
```